errors/Makefile.am

```diff
@@ -24,6 +30,12 @@ EXTRA_DIST = \
 	template.am \
 	aliases alias-link.sh alias-upgrade errorpage.css TRANSLATORS COPYRIGHT
 
+if ENABLE_TRANSLATION
+ALIAS_LINKER=$(SHELL) $(srcdir)/alias-link.sh "$(LN)" "$(RM)" "$(DESTDIR)$(DEFAULT_ERROR_DIR)"
+else !ENABLE_TRANSLATION
+ALIAS_LINKER=$(TRUE)
+endif
```

Comment:
We can merge the two `if ENABLE_TRANSLATION` blobs together by moving this alias-link.sh blob up. I am not sure whether dedicated (to a single variable/program/concern) blobs work better.